### ISSUE TITLE
change markers for versioning systems such as sd should be detected.

### ISSUE THREAD

**Issue by nojvek** [Microsoft/TypeScript#8448]
Sd change markers should be detected by typescript and have warnings just the way it detects issues for git change markers.

**Expected behavior:**
Have warnings

**Actual behavior:** 
No warnings. Errors are ambigious.

e.g 
```
>>>> ORIGINAL accessibilityTab.ts#2 ++++++++++++++++++++++++++++++++++++++++++++++++
console.log("y0")
==== THEIRS accessibilityTab.ts#3 ++++++++++++++++++++++++++++++++++++++++++++++++
console.log("boo")
==== YOURS accessibilityTab.ts ++++++++++++++++++++++++++++++++++++++++++++++++
console.log("yay")
<<<< END ++++++++++++++++++++++++++++++++++++++++++++++++
```

results in

```
 error TS1109: Expression expected.
 error TS1109: Expression expected.
 error TS1109: Expression expected. 
 error TS1109: Expression expected.
 ...
```

**Comment by mhegazy** [comment]
If you are interested to send us a PR feel free to. I do not think anyone one core team will be doing this anytime soon, given that no one outside MS knows about sd.exe, and even MS is moving to git.

**Comment by basarat** [comment]
TIP: one can capture them as conflict markers in the scanner : https://github.com/Microsoft/TypeScript/blob/cb9be669ec79296ea2e4b42fb3bdbe4660ca6bca/src/compiler/scanner.ts#L514-L568 :rose:

**Comment by nojvek** [comment]
I did some research. It seems the most used conflict marker are 7 characters. Git, svn, mercurial all use '<<<<<<<' (7) only sd uses '<<<<' (4). So 7 seems like an official standard. I am closing this because sd has a very small market share (only some projects inside MS). I'd rather file a bug on sd to have 7 chars in conflict markers.
